java/serving/src/main/java/feast/serving/config/ServerModule.java

```diff
@@ -0,0 +1,45 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ * Copyright 2018-2021 The Feast Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package feast.serving.config;
+
+import com.google.inject.AbstractModule;
+import feast.serving.grpc.OnlineServingGrpcServiceV2;
+import io.grpc.Server;
+import io.grpc.ServerBuilder;
+import io.grpc.protobuf.services.ProtoReflectionService;
+import io.opentracing.contrib.grpc.TracingServerInterceptor;
+
+public class ServerModule extends AbstractModule {
+
+  @Override
+  protected void configure() {
+    bind(OnlineServingGrpcServiceV2.class);
+  }
+
+  //  @Provides
+  public Server provideGrpcServer(
+      OnlineServingGrpcServiceV2 onlineServingGrpcServiceV2,
+      TracingServerInterceptor tracingServerInterceptor) {
+    ServerBuilder<?> serverBuilder = ServerBuilder.forPort(6566);
```

Comment:
Port should be probably pulled from properties

Reply:
Yeah good point